SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3531,6 +3540,64 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 	return false;
 }
 
+void USpatialNetDriver::ServerReplicateActors_BuildConsiderList(TArray<FNetworkObjectInfo*>& OutConsiderList, const float ServerTickTime)
+{
+	const bool bSpatialNetworking = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+
+	if (!bSpatialNetworking || AuthorityDebugger == nullptr)
+	{
+		// Default case to only consider auth actors for replication
+		Super::ServerReplicateActors_BuildConsiderList(OutConsiderList, ServerTickTime);
+		return;
+	}
+
+	// Spatial authority debugger case to check non-auth actors for invalid changes and then add auth actors to the consider list for
+	// replication
+	TArray<FNetworkObjectInfo*> TmpConsiderList;
```

Comment:
speculative, but it could be faster to use OutConsiderList and prune using RemoveAll or Swap (if order isn't important, which it might be).